src/styles/style.css

```diff
@@ -0,0 +1,250 @@
+body {
+  font-family: Roboto, sans-serif;
+  margin: 0;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header__logo-img {
+  display: flex;
```

Comment:
The '.header__logo-img' class has 'display: flex;' applied, which is unnecessary for an image element unless you intend to use flex properties on it. Consider removing it if not needed.